temperature/get_weather.py

```diff
@@ -0,0 +1,22 @@
+import os
+from typing import Any
+import aiohttp
+
+
+async def get_weather(city_name: str) -> Any:
+    URL = "http://api.weatherapi.com/v1/current.json"
+    API_KEY = os.environ.get("API_KEY")
+    PARAMS = {"key": API_KEY, "q": city_name}
+
+    async with aiohttp.ClientSession() as session:
+        async with session.get(URL, params=PARAMS) as response:
+            data = await response.json()
+
+            if response.status == 200:
+                weather_data = data["current"]
+                temperature = weather_data["temp_c"]
+                return temperature
+
+            raise ValueError(
+                f"Failed to fetch temperature for {city_name}."
+            )
```

Comment:
what about dependency injection?
Also I'm not sure this function should now api_key, even though it's getting from env.